src/Models/ProjectLabels.coffee

```diff
@@ -0,0 +1,31 @@
+BaseModel = require '../BaseModel'
+Utils = require '../Utils'
+
+class ProjectLabels extends BaseModel
+  list: (projectId, params = {}, fn = null) =>
```

Comment:
Yes we need to change ProjectIssues but we should do that in a seperate pull request.